### DISCUSSION TITLE
After installation, test email stuck in queue (pending)

### DISCUSSION THREAD

**Discussion by jazer143** [postalserver/postal#1991] [Help with using Postal]
Hi Expert, I'm a newbie, how to fix this issue, after installation and testing stucked in queue (pending). Thank for helping.

**Comment by Akazox** [comment]
same issue

**Comment by willpower232** [comment]
What is the output of `postal status`? Is everything running or did some things fail to start?

**Reply by Akazox** [reply]
root@mail:~# postal status
NAME                COMMAND                  SERVICE             STATUS              PORTS
postal-cron-1       "/docker-entrypoint.…"   cron                running
postal-requeuer-1   "/docker-entrypoint.…"   requeuer            exited (1)
postal-smtp-1       "/docker-entrypoint.…"   smtp                running
postal-web-1        "/docker-entrypoint.…"   web                 running
postal-worker-1     "/docker-entrypoint.…"   worker              exited (1)

**Reply by willpower232** [reply]
if those two are exited then that would explain everything, its likely that the rabbit mq was not ready when postal was, try `postal start` again and see if everything starts working
